resource/resource.go

```diff
@@ -43,17 +46,22 @@ var (
 	}
 
 	mandatorySourceKeys = map[string]struct{}{
-		"owner":        {},
-		"repo":         {},
-		"access_token": {},
+		"owner":      {},
+		"repo":       {},
+		access_token: {},
```

Comment:
For my own education, why do you have to keep the quotes form the first two? And same question for the first 3 elements of `optionalSourceKeys` below?

Reply:
Ah. I see. It's really a petty they decided that the good old convention of naming constant in UPPERCASE was too clear and unambiguous to be preserved. :disappointed:

Having a mix is IMO even more confusing. Having constant for all would be much better.

Reply:
Done. It has been a pain, but it is now done once and for all :-)